src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -354,6 +354,10 @@ private ClassDeclNode ParseSystemType(Type type, string alias)
                     CLRModuleType.GetInstance(interf, Module, string.Empty);
                 }
             }
+            if(classnode.IsInterface)
```

Comment:
Are you able to track down why this got regressed? I do not see this check existed in previous version of code